Bottom: 32efda14471bed8fe0a19146614e0fa6e2309114
Top:    38f85db4f49c60edcec21574f1df61168e36922e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 13:51:53 +0000

Refresh of remove-debug-from-hairline

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 7c600ac..8f6f681 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2845,8 +2845,6 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
 
   *res = NULL;
 
-  g_warning ("BEGIN BOOLEAN");
-
   /* Make copies for tracking polygon parentage (DEBUG) */
   if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
       return err_no_memory;
@@ -2971,7 +2969,6 @@ out:
       M_Set_Parentage (*res, parentage);
     }
 
-  g_warning ("END BOOLEAN");
   return code;
 }				/* poly_Boolean_free */
